na_protein_fastq.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import modules.dna_rna_tools as dna_rna_tools
```

Comment:
по PEP8 импорты из стандартной библиотеки и своей надо разделять одним пробелом
```suggestion
import sys

import modules.dna_rna_tools as dna_rna_tools
```

Здорово, что после всех импортов ты поставила две пустые строки, по PEP8!